docs/concepts/json_schema.md

```diff
@@ -650,6 +650,8 @@ print(json.dumps(ta.json_schema(), indent=2))
 """
 ```
 
+# TODO: this is no longer supported... do we need it to be?
+
```

Comment:
So, I'd like to chat more with @dmontagu about this, but basically this means we don't support a `json_schema_extra` callable applying AFTER a `json_schema_extra` dict update.

I don't think we should support this, anyways. I think it was a mistake in the first place, and I don't think this is a pattern used by many people at all.

If we make this release and people are really upset about this (I really don't think anyone will be), I'd be happy to hold the responsibility of fixing in a patch.

This documentation example is the only test that's failing with these changes.

Reply:
Update: chatted about this. I've added a warning that this is no longer supported, and if folks want this old (only have supported behavior), they can open an issue with a request and explanation of their use case.

Thus, marking as resolved.